wiki/osu!tourney/osu!lazer_tournament_client/en.md

```diff
@@ -0,0 +1,66 @@
+# osu!lazer tournament client
+
+**osu!lazer tournament client** is the official client that overlays osu!tourney.
+
+Users encountering problems with the client may [create an issue](https://github.com/ppy/osu/issues) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
+
+## Setup
+
+To start the osu!lazer tournament client, you need to specify a launch argument to the executable.
+
+To do this, create a new shortcut on the desktop and set the location for it to `%LOCALAPPDATA%\osulazer\osu!.exe --tournament`. This will make that specific shortcut start up osu!lazer in the tournament client mode.
+
+Because the osu!lazer tournament client is only an overlay for osu!tourney, it will need to be set up as well. To make sure osu!tourney will work with osu!lazer you will need to set the Release stream in osu!tourney to Cutting Edge (Experimental) and create an empty file called `ipc.txt` inside the installation folder of osu!tourney. After that follow the [osu!tourney setup](/wiki/osu!tourney/Setup).
```

Comment:
```suggestion
Because the osu!lazer tournament client is only an overlay for osu!tourney, that will need to be set up as well. Set the Release stream in osu!tourney to `Cutting Edge (Experimental)` and create an empty file called `ipc.txt` inside the installation folder of osu!tourney. After that, follow the [osu!tourney setup guide](/wiki/osu!tourney/Setup).
```